.github/workflows/build_samples.yml

```diff
@@ -0,0 +1,49 @@
+name: Build Samples
+
+on: [push, pull_request]
+
+jobs:
+  windows:
+    name: Windows
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
I consider name is still "bad"; see https://github.com/XboxDev/nxdk/pull/168#discussion_r352331736.

*(Extra discussion as old discussion has diverged; with 2/3 items resolved)*